examples/1-sklearn-tree-aws/deploy.yaml

```diff
@@ -0,0 +1,7 @@
+mlctl_version: 0.1
+metadata:
+  name: my-new-endpoint
```

Comment:
I think this is a deployment instance and should have an instance name, e.g. deploy_weight_sklearn